autoload/neoterm/repl.vim

```diff
@@ -24,12 +24,12 @@ function! neoterm#repl#term(id)
   if has_key(g:neoterm.instances, a:id)
     let g:neoterm.repl.instance_id = a:id
     let g:neoterm.repl.loaded = 1
-
+    call add(g:neoterm_repl_command, g:neoterm_eof)
```

Comment:
reading the code again, I think we can handle that inside the `neoterm#repl#set`.

Reply:
Yeah, that's right.